src/fetch_reply.py

```
@@ -478,10 +526,60 @@ def create_threaded_reply_draft(self, original_message_id, reply_text, from_acco
+                                    # Still return draft_id since creation succeeded - let caller decide
+                                    return draft_id
+                            elif verify_response.status_code == 404:
+                                logger.error(f"Draft created but NOT FOUND in Graph API (404) - draft may have been deleted: {draft_id}")
+                                # Draft doesn't exist - return None to prevent storing invalid draft_id
+                                return None
+                            else:
+                                logger.warning(f"Draft created but verification returned {verify_response.status_code} - returning draft_id anyway (may be temporary issue)")
+                                # Return draft_id since creation succeeded - verification failure might be temporary
+                                return draft_id
+                        except httpx.TimeoutException:
+                            logger.warning(f"Draft created but verification timed out - returning draft_id anyway (may be temporary network issue)")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)